packages/react/src/components/ListBoxBase/ListBoxBase.styles.ts

```diff
@@ -18,4 +18,24 @@ export const useStyles = css({
     overflow: 'visible',
     width: 'auto',
   },
+
+  '.manifest-listbox-no-results-container': {
```

Comment:
These classnames use [BEM](https://getbem.com/naming/) convention, so this would be `manifest-listbox__no-results-container` and similarly `.manifest-listbox__loading-container` below